src/main/java/core/basesyntax/service/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ShopService;
+import java.util.Map;
+
+public class ShopServiceImpl implements ShopService {
+    private static final String separator = ",";
+
+    @Override
+    public String makeReport() {
```

Comment:
where do you form 1st line?
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/8d94ab33-aa68-4f8e-b178-20ffc5db5a01)